openmdao/docs/openmdao_book/features/features.ipynb

```diff
@@ -77,7 +77,9 @@
     "\n",
     "- [Determining Variable Shapes at Runtime](experimental/dyn_shapes.ipynb)\n",
     "- [Simultaneous Coloring of Approximated Derivatives](experimental/approx_coloring.ipynb)\n",
-    "- [Working with Plugins](experimental/plugins.ipynb)\n"
+    "- [Working with Plugins](experimental/plugins.ipynb)\n",
+    "- [Jax for ExplicitComponents using the JaxExplicitComponent class](experimental/jax_explicitcomp_api.ipynb\n",
```

Comment:
missing closing paren here, extra in the next line

Reply:
fixed